src/ident/MessageId.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1); // @codeCoverageIgnore
+
+namespace Rinq\Ident;
+
+/**
+ * MessageID uniquely identifies a message that originated from a session.
```

Comment:
I know this is pedantic, but check the casing - `MessageID` vs `MessageId`.  When talking about the class, use `MessageId`.  When used in a sentence, use "message ID".